src/app/components/FoodCardList/index.tsx

```diff
@@ -0,0 +1,18 @@
+import FoodCard, { FoodCardProps } from '../FoodCard';
+import styles from './styles.module.css';
+
+export interface FoodCardListProps {
+  title: string;
+  foodCards: FoodCardProps[];
+}
+
+export default function FoodCardList({ title, foodCards }: FoodCardListProps) {
+  return (
+    <section className={styles.foodCardList}>
+      <h2 className={styles.foodCardListTitle}>{title}</h2>
+      {foodCards.map((food) => (
```

Comment:
```suggestion
      {foodCards?.map((food) => (
```
para evitar problemas